client/apps/chrome-extension/src/contents/page.tsx

```diff
@@ -85,7 +87,7 @@ function Content() {
       }}
       name={`deskroom-${process.env.NODE_ENV}`}>
       <DeskroomUserProvider>
-        <Theme>
+        <Theme accentColor="iris">
```

Comment:
```suggestion
        <Theme>
```
accent color 바꾸면 다른 컴포넌트들에 영향을 줄거 같은데 이유가 있을까요?

Reply:
해당 부분은 selection color가 지정되지 않아 작업했습니다!
selection color 문제는 해결했으니 제외하도록 하겠습니다.

Reply:
아래 코멘트에 달아두었습니다!
제 생각엔 결국 하나의 accent color를 지정해두는 것이 장기적으로 나은 방법인 것 같습니다.
(다만 지금은 스타일 작업에 너무 큰 코스트가 들어가니 selection만 지정해두는 것으로 작업을 남겨두도록 하겠습니다.)